Sources/Authenticator/Views/Primitives/TextField.swift

```diff
@@ -87,7 +87,7 @@ struct TextField: View {
         let textView = SwiftUI.Text(label)
             .foregroundColor(theme.colors.foreground.disabled.opacity(0.6))
             .font(theme.fonts.body)
-        textView.accessibilityHidden(true)
+        _ = textView.accessibilityHidden(true)
         return textView
```

Comment:
same suggestion as in another file: can this be chained with `accessibilityHidden` modifier?